tensorflow/examples/skflow/resnet.py

```diff
@@ -152,6 +153,10 @@ def res_net(x, y, activation=tf.nn.relu):
       model_fn=res_net, n_classes=10, batch_size=100, steps=100,
       learning_rate=0.001, continue_training=True)
 
+classifier = learn.TensorFlowEstimator(
```

Comment:
you should remove previous lines and `save` call - fit now auto-saves/restores into logdir.